ocaml/xapi/xapi_vtpm.ml

```diff
@@ -53,7 +53,7 @@ let introduce ~__context ~vM ~persistence_backend ~contents ~is_unique =
   ref
 
 (** Contents from unique vtpms cannot be copied! *)
-let get_contents ~__context ?from () =
+let copy_or_create_contents ~__context ?from () =
```

Comment:
Looking from the outside, the semantics are a bit surprising. We either get a copy of an existing secret, or an empty secret. The name captures this (maybe `copy_or_create_secret`) but it's hard to imagine a client that finds this useful.

Reply:
This function is not exposed in the interface, it's meant to be used inside the module only